dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -1,64 +1,117 @@
 #!/usr/bin/env python
 
 import sys
+import copy
+import Queue
 import redis
 import subprocess
 import syslog
-from swsssdk import ConfigDBConnector
+import os
+from swsscommon import swsscommon
 
-class BGPConfigDaemon:
 
-    def __init__(self):
-        self.config_db = ConfigDBConnector()
-        self.config_db.connect()
-        self.bgp_asn = self.config_db.get_entry('DEVICE_METADATA', 'localhost')['bgp_asn']
-        self.bgp_neighbor = self.config_db.get_table('BGP_NEIGHBOR')
+def run_command(command):
+    syslog.syslog(syslog.LOG_DEBUG, "[bgp cfgd] execute command {}.".format(command))
+    p = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
+    stdout = p.communicate()[0]
+    p.wait()
+    if p.returncode != 0:
+        syslog.syslog(syslog.LOG_ERR, '[bgp cfgd] command execution returned {}. Command: "{}", stdout: "{}"'.format(p.returncode, command, stdout))
+
+
+class BGPConfigManager(object):
+    def __init__(self, daemon):
+        self.daemon = daemon
+        self.bgp_asn = None
+        self.bgp_message = Queue.Queue(0)
+        daemon.add_manager(swsscommon.CONFIG_DB, swsscommon.CFG_DEVICE_METADATA_TABLE_NAME, self.__metadata_handler)
+        daemon.add_manager(swsscommon.CONFIG_DB, swsscommon.CFG_BGP_NEIGHBOR_TABLE_NAME, self.__bgp_handler)
+
+    def __metadata_handler(self, key, op, data):
```

Comment:
Have you actually tested this part? For sure it wasn't working before and I have my doubts that it's working now.

Reply:
Yes, it doesn't work and I will discuss with Guohan about it.

Reply:
@Pterosaur bgpcfgd was introduced by https://github.com/Azure/sonic-buildimage/pull/2870/ and it is not working code in addition to causing other issues. It continues to be non working code and I have called for a meeting to discuss this. Please do not merge any changes in the area until the meeting has taken place. cc @lguohan @xinliu-seattle @zhenggen-xu @MichelMoriniaux @mslocrian @jipanyang

Reply:
remove this bgp_asn handler, it is not used and not needed.

Reply:
In current version,  bgp_asn handler is used to set the ASN, otherwise we need a synchronized query , something like a `while true polling` , to get the ASN. Because we don't known when the ASN was set.
Can I just remove these commands and add a TODO comment?

Reply:
yes